### ISSUE TITLE
Make website redirect http to https

### ISSUE THREAD

**Issue by ShaddyDC** [olivierkes/manuskript#1040]
While there isn't any valuable secret data on the website, using https is a good practice and, nowadays, its absence feels sketchy. The site already supports it, so why not go the rest of the way of making it the default? 
I drop by this project every couple of months, and end up on the http page every time after using the http link in the GitHub about section (http://www.theologeek.ch/manuskript). Changing just that link would be enough to make me happy, but I imagine many links still point to the http site, and you'd want those users to end up on https too.

**Comment by TheJackiMonster** [comment]
Yes, you are right. I probably never noticed since my web browsers always switch to https by default if available. ^^'

Technically it doesn't matter to much since the download links are all pointing to Github using https. So at least there shouldn't be any security wise problem because of those links.

**Comment by ShaddyDC** [comment]
It also uses https automatically for me after having loaded it once and knowing it's available, on both chrome and firefox, which could also explain it. I'm not sure what exactly causes that, and it has its advantages, but it can also lead to oversights it seems :P

**Comment by TheJackiMonster** [comment]
The README.md got adjusted now to use only https links. Should I check the wiki as well?

**Comment by ShaddyDC** [comment]
The main link that tripped me up was this one:
![Screenshot_20220618_115102](https://user-images.githubusercontent.com/18403034/174432512-6f3a5caa-5e68-4a82-a7dd-e1f46fba2d22.png)

It would be good if the wiki and the website itself only used https links, so an attacker wouldn't be able to hijack a connection, though I guess this wouldn't be a primary target.
Does the website redirect to https now? It does for me, but that might just be because I've already used the https site.

**Comment by TheJackiMonster** [comment]
The website doesn't redirect to https automatically. I'm not sure if I have the permission to install something like this. But I will adjust the links on Github I can find.

**Comment by ShaddyDC** [comment]
That's a shame. Thank you for your effort, though! :)

**Comment by TheJackiMonster** [comment]
> The main link that tripped me up was this one: ![Screenshot_20220618_115102](https://user-images.githubusercontent.com/18403034/174432512-6f3a5caa-5e68-4a82-a7dd-e1f46fba2d22.png)
> 
> It would be good if the wiki and the website itself only used https links, so an attacker wouldn't be able to hijack a connection, though I guess this wouldn't be a primary target. Does the website redirect to https now? It does for me, but that might just be because I've already used the https site.

I think, I can't change that link since @olivierkes is still owner of the repository on Github. So I don't have the permission to adjust the description unfortunately. Maybe he can adjust it. ^^'

**Comment by 20marcmader20** [comment]
Am 18. Juni 2022 20:15:04 MESZ schrieb Tobias Frisch ***@***.***>:
>The website doesn't redirect to https automatically. I'm not sure if I have the permission to install something like this. But I will adjust the links on Github I can find.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/olivierkes/manuskript/issues/1040#issuecomment-1159530003
>You are receiving this because you are subscribed to this thread.
>
>Message ID: ***@***.***>
-- Gesendet von /e/-Mail.
